### PR TITLE
Don't show "Deployed" for plain RCs in web console

### LINKED ISSUE #6200: Plain RC status shows as "Deployed" in web console
This might be working as intended, but it's odd to see status "Deployed" for a replication controller that is not a deployment.

openshift_web_console

### PR DESCRIPTION
**Pull request by spadgett**
Don't show status "Deployed" for replication controllers that don't have a deployment config.

Fixes #6200

### CODE REVIEW

File: assets/app/views/deployments.html [hunk 1]
```diff
@@ -116,8 +116,7 @@
                         <a ng-href="{{deployment | navigateResourceURL}}">{{deployment.metadata.name}}</a>
                       </td>
                       <td>
```
Comment (jwforres) [inline assets/app/views/deployments.html]: the RCs table is missing its mobile data-title attributes

**Comment by spadgett** [comment]
@jwforres Updated to add data-title.

**Comment by jwforres** [comment]
going to go ahead and [merge] this since the headers were missing at mobile

**Comment by spadgett** [comment]
[merge]

**Pull request merged**